### PR TITLE
Fixed README.

### PR DESCRIPTION
**Pull request by elonmusksama**
Fixed some typos and formatting issues, also linked some missing Staff Github profiles.

### CODE REVIEW

**Comment by elonmusksama** [comment]
@leijurv

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Impact Client
 
-Impact is an advanced Minecraft Utility Mod aimed towards user friendliness.
+Impact is an advanced Minecraft Utility Mod aimed towards user-friendliness.
```
Comment (l1ving) [inline README.md:3]: no

File: README.md [hunk 2]
```diff
@@ -9,22 +9,22 @@ You can [download Impact][download] by visiting the Impact Website.
 
 ## Help and Support
 
-If you need help using Impact, you should instead ask for it on the [discord] server. Use the `#help` channel so that you don't clutter up `#general` with support questions. Also make sure to check `#announcements`, pinned messages and use the search tool to see if your question has already been answered.
+If you need help using Impact, you should instead ask for it on the [discord] server. Use the `#help` channel so that you don't clutter up `#general` with support questions. Make sure to check `#announcements`, pinned messages and use the search tool to see if your question has already been answered.
```
Comment (l1ving) [inline README.md:12]: there was nothing wrong with this

File: README.md [hunk 3]
```diff
@@ -9,22 +9,22 @@ You can [download Impact][download] by visiting the Impact Website.
...
 
-It is also recommended to check the wiki first, it may have your fix:
-[Common Issues](https://github.com/ImpactDevelopment/ImpactIssues/wiki/Common-Issues-FAQ)
+It is also recommended checking the wiki first, it may have your fix:
+[Common Issues][Common Issues].
```
Comment (l1ving) [inline README.md:15]: this is only linked once, no sense in changing it

File: README.md [hunk 4]
```diff
@@ -9,22 +9,22 @@ You can [download Impact][download] by visiting the Impact Website.
...
 
 ## Bugs
 
-Please report any bugs you find [here][issues]. Try to be as descriptive as possible and make sure to check if the bug has already been reported _before_ reporting it youself. Please only report one bug per issue - if you have encountered multiple issues, report multiple bugs as seperate    issues.
+Please report any bugs you find [here][issues]. Try to be as descriptive as possible and make sure to check if the bug has already been reported _before_ reporting it yourself. Please only report one bug per issue - if you have encountered multiple issues, report multiple bugs as separate issues.
```
Comment (l1ving) [inline README.md:19]: ok

File: README.md [hunk 5]
```diff
@@ -9,22 +9,22 @@ You can [download Impact][download] by visiting the Impact Website.
...
 
 ## Feature Requests
 
-New features should also be reported as [issues]. Please try to describe the feature you are suggesting in as much detail as possible and explain _why_ it would be useful. As explained above, please first check the feature hasn't already been requested and make sure your issue only covers one feature. If you have multiple ideas, open seperate issues.
+New features should also be reported as [issues]. Please try to describe the feature you are suggesting in as much detail as possible and explain _why_ it would be useful. As explained above, please first check the feature hasn't already been requested and make sure your issue only covers one feature. If you have multiple ideas, open separate issues.
 
 ## Credit
 
-   Impact is developed by [ZeroMemes], [leijurv] and [LeafHacker]. It is built on top of [ClientAPI] and supports [Baritone] which is developed by [leijurv]
+Impact is developed by [ZeroMemes], [leijurv] and [LeafHacker]. It is built on top of [ClientAPI] and supports [Baritone], which is developed by [leijurv].
```
Comment (l1ving) [inline README.md:27]: This was probably intended for a reason

**Review by l1ving — CHANGES_REQUESTED** [review]

**Comment by elonmusksama** [comment]
shut up bella

**Comment by elonmusksama** [comment]
:KILLHER:

File: README.md (hunk 5 above)
Comment (elonmusksama) [reply README.md:27]: stay mad

File: README.md (hunk 4 above)
Comment (elonmusksama) [reply README.md:19]: yes

File: README.md (hunk 3 above)
Comment (elonmusksama) [reply README.md:15]: i wanted to keep the same md style, its much cleaner dis way

File: README.md (hunk 2 above)
Comment (elonmusksama) [reply README.md:12]: dont care didnt ask 😎

File: README.md (hunk 1 above)
Comment (elonmusksama) [reply README.md:3]: the correct way 😎

**Comment by leijurv** [comment]
why are mommy and daddy fighting on github 🥺

File: README.md (hunk 1 above)
Comment (leijurv) [reply README.md:3]: Screen Shot 2020-12-05 at 12 57 12 PM

File: README.md (hunk 2 above)
Comment (leijurv) [reply README.md:12]: i like don't care either way so im going to click RESOLVE CONVERSATION

troll face emoji sunglasses emoji cat smirk emoji

File: README.md (hunk 5 above)
Comment (leijurv) [reply README.md:27]: not that i can see so i think it's fine idk

File: README.md (hunk 1 above)
Comment (l1ving) [reply README.md:3]: trolled

**Comment by elonmusksama** [comment]
I'm sowwy weijurv 😿

**Comment by elonmusksama** [comment]
pwease merge the PR :333

**Comment by DoctorBang** [comment]
This is such a user-friendly conversation.

**Comment by elonmusksama** [comment]
yay thank you 0x 22

**Pull request merged**